### ISSUE TITLE
Can not deploy static website service with patch for the second time

### ISSUE THREAD

**Issue by sumar-triniti** [aws/copilot-cli#5526]
I have static site deployed with copilot, and I have following patch. First time I run copilot svc deploy it work, and i can confirm the patch is work. 
```
- op: add
  path: /Resources/CloudFrontDistribution/Properties/DistributionConfig/CacheBehaviors
  value: 
    - PathPattern: "/anotherfiles/*"
      Compress: true
      AllowedMethods: ["GET", "HEAD"]
      ViewerProtocolPolicy: redirect-to-https
      CachePolicyId: 658327ea-f89d-4fab-a63d-7e88639e58f6 # See https://go.aws/3bJid3k
      TargetOriginId: !Sub 'anotherfile-id'
```

But when i run svc deploy again I got an error:
```
- No new infrastructure changes for stack myapp-development-mysvc
deploy service: change set with name copilot-838a5dcd-2f71-408d-93e1-3ee41ee8483e for stack myapp-development-mysvc has no changes: Properties validation failed for resource CloudFrontDistribution with message:
#/DistributionConfig/CacheBehaviors: expected type: JSONArray, found: JSONObject
```

Copilot version copilot-linux-amd64-v1.32.0

**Comment by Lou1415926** [comment]
@sumar-triniti Hey there 👋🏼 ! This should be the same bug as reported [here](https://github.com/aws/copilot-cli/issues/5472#issuecomment-1818341925). My teammate @KollaAdithya fixed it by https://github.com/aws/copilot-cli/pull/5498 though it's not released yet. 

Fortunately you don't have to wait for the release!

If you have `jq` installed on your machine, you can run:
```console
AWS_REGION=<env-region> aws resourcegroupstaggingapi get-resources \
  --tag-filters "Key=copilot-application,Values=<your application name>" "Key=aws:cloudformation:logical-id,Values=PipelineBuiltArtifactBucket" \
  --resource-type-filters s3:bucket | jq .ResourceTagMappingList | jq '.[0]'.ResourceARN
```

This should give you something like `"arn:aws:s3:::stackset-<app-name>-inf-pipelinebuiltartifactbuc-<random string>"`. Note the ARN value down.

For now, you can use YAML patch to get around the bug:
```yaml
- op: replace
  path: "/Resources/CopyAssetsStateMachineRole/Properties/Policies/0/PolicyDocument/Statement/1/Condition/StringEquals/kms:EncryptionContext:aws:s3:arn"
  value: "<The ARN that we noted down>" # [!]The quotation marks are necessary
```

If you don't have `jq`, you could still run 
```
AWS_REGION=<env-region> aws resourcegroupstaggingapi get-resources \
  --tag-filters "Key=copilot-application,Values=<your application name>" "Key=aws:cloudformation:logical-id,Values=PipelineBuiltArtifactBucket" 
```
which _should_ give you only one bucket. You can then just look for the ARN of that bucket. What `jq` did was really just parsing the JSON for you.

Sorry for the trouble!! Once the fix is released, you can remove the yaml patch snippet.

**Comment by sumar-triniti** [comment]
Glad to hear that this issue is fixed, can not wait for next release.

I really appreciate for your workaround to unblock us.

Thanks!

**Comment by iamhopaul123** [comment]
Hello! The fix is now released in v1.32.1: https://github.com/aws/copilot-cli/releases/tag/v1.32.1
